src/Components/Auth/LoginForm.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from 'react';
+import {
+  Button,
+  InputGroup,
+  Modal,
+  ModalBody,
+  ModalFooter,
+  Alert,
+  FormControl
+} from 'react-bootstrap';
+import {FiMail, FiLock} from 'react-icons/fi';
```

Comment:
Anh thấy nó hướng dẫn toàn /fa, /fi này là một bộ icon khác à?

Reply:
Vâng. có 2 bộ. A dùng bộ nào thì a dùng

Reply:
Thật ra là còn rất nhiều bộ. 2 bộ này chỉ là 2 trong số nhiều bộ đó thôi.
ví dú như: di, gi, go, io

Nhưng các chữ viết tắt này đều có ý nghĩ hết nhé

Reply:
à ok, tại mới thấy bộ /fa, ok Thanks